ReactViewResources/Loader.ts

```diff
@@ -117,34 +124,38 @@ export function loadStyleSheet(stylesheet: string): void {
     innerLoad();
 }
 
-export function loadPlugins(plugins: string[][], mappings: Dictionary<string>): void {
+export function loadPlugins(plugins: string[][]): void {
     async function innerLoad() {
         try {
             await BootstrapTask.promise;
 
-            if (mappings) {
-                let paths = Object.assign(getRequirePaths(), mappings);
-                require.config({
-                    paths: paths
-                });
-            }
-
             if (plugins && plugins.length > 0) {
                 // load plugin modules
                 let pluginsPromises: Promise<void>[] = [];
                 plugins.forEach(m => {
-                    const ModuleName = m[0];
-                    const NativeObjectName = m[1];
+                    const ModuleName: string = m[0];
+                    const NativeObjectName: string = m[1];
+                    const MainModule: string = m[2];
```

Comment:
ModuleName and MainModule ... whats the difference?

Reply:
ModuleName => "ExampleView.view";
MainSource => "/Example/Generated/ExampleView.view.js";

I'll update it to MainJavascriptSource